server/api/routes.ts

```diff
@@ -220,47 +259,47 @@ function pack(promise, res, next) {
     });
 }
 
-function getUser(req) {
+async function getUser(req) {
   // Get username
-  return LDAP.getActiveUser(req);
+  let temp = await LDAP.getActiveUser(req);
+  return temp;
```

Comment:
No need for a temp variable here; you can `return await`

Reply:
This was something I was testing and forgot to reverse. Thanks for the catch, will fix.